src/workerd/api/node/util.c++

```diff
@@ -240,4 +241,30 @@ jsg::JsValue UtilModule::getBuiltinModule(jsg::Lock& js, kj::String specifier) {
   return js.undefined();
 }
 
+void UtilModule::processExitImpl(jsg::Lock& js, int code) {
```

Comment:
node.js stringifies every code. for example running `process.exit('00')` or `process.exit('11')` works. let's make sure we accept any input.

Reply:
I believe there are already tests elsewhere that verify that the value coercion works with the jsg type mappings. Manually checking `process.exit('00')` with this and it works as expected.